content/doc/book/pipeline/getting-started.adoc

```diff
@@ -22,106 +23,198 @@ This section introduces some of the key concepts to Jenkins Pipeline and
 help introduce the basics of defining and working with Pipelines inside of a
 running Jenkins instance.
 
+
 == Prerequisites
 
 To use Jenkins Pipeline, you will need:
 
 * Jenkins 2.x or later (older versions back to 1.642.3 may work but are not
   recommended)
-* Pipeline plugin
-footnoteref:[pipeline, link:https://plugins.jenkins.io/workflow-aggregator[Pipeline plugin]]
+* Pipeline plugin,
+  footnoteref:[pipeline,link:https://plugins.jenkins.io/workflow-aggregator[Pipeline plugin]]
+  which is installed as part of the "suggested plugins" (specified when running
+  through the <<../installing#setup-wizard,Post-installation setup wizard>>
+  after <<../installing#,installing Jenkins>>).
+
+Read more about how to install and manage plugins in
+<<../managing/plugins#,Managing Plugins>>.
 
-To learn how to install and manage plugins, consult <<../managing/plugins#, Managing Plugins>>.
 
 == Defining a Pipeline
 
-Scripted Pipeline is written in
-link:http://groovy-lang.org/[Groovy].
-The relevant bits of
-link:http://groovy-lang.org/semantics.html[Groovy syntax]
-will be introduced as necessary in this document, so while an understanding of
-Groovy is helpful, it is not required to work with Pipeline.
+Both Declarative and Scripted Pipeline is written in
+link:http://groovy-lang.org/[Groovy] syntax. Relevant components of
```

Comment:
I think it might help others understand if we remind them very early that both Pipeline variants are domain specific languages.  Scripted Pipeline uses groovy syntax, while declarative pipeline doesn't really use groovy syntax.

When script blocks are used inside a declarative pipeline, I think of those blocks as Scripted Pipeline embedded in a Declarative Pipeline.

So, instead of:

> Both Declarative and Scripted Pipeline is written in Groovy syntax

you might consider

> Declarative Pipeline and Scripted Pipeline are both domain-specific languages to describe portions of a software delivery pipeline.
> Scripted Pipeline is written in a limited form of [Groovy](http://groovy-lang.org/) syntax.

The "limited form" phrase is intentionally included because there are Groovy constructs which the Scripted Pipeline does not support.  Groovy purists are frequently perplexed when they can't use certain groovy syntax elements in their Scripted Pipeline.

Reply:
Excellent points!  I've been saying that Pipeline uses its own DSL which is based on Groovy syntax.  But it sounds like that might be proper for Declarative and use "limited form of Groovy syntax" when discussing Scripted.

Reply:
Thanks very much @MarkEWaite .

That was somewhat my fault. The original text was:

> Scripted Pipeline is written in Groovy.

... and I wanted to integrate Declarative Pipeline into this part of the discussion, so thanks for clarifying this situation with respect to Scripted/Declarative Pipeline syntax.

Personally, I didn't find the original statement that helpful because to relative newbies (like myself) who are reading about these concepts from this documentation, I kept wondering 'Is Scripted Pipeline written up in Groovy syntax, or was Scripted Pipeline's functionality developed in the Groovy language itself, or both?'

Anyway, I'll use your suggestion as it clarifies the situation and avoids this potential confusion/distraction.

Thanks also @StackScribe , we mention at least 3 times on the previous ("Pipeline" chapter) page about Pipeline being its own DSL ;-) I'm almost getting to the point where I can't see the forest for the trees anymore but new readers of this page should hopefully pick this up lol

Reply:
@MarkEWaite , I ended up re-writing your first sentence to:

> As mentioned on this chapter’s initial Pipeline page, both Declarative and Scripted Pipeline are DSLs [1] to describe portions of a continuous delivery pipeline for your software.

where "Pipeline page" links back to the "Pipeline" chapter page.

Reply:
FYI @MarkEWaite and @StackScribe , I've modified the `[TIP]` (about syntax highlighting) further down this page to suit the changes above.

Let me know if the new wording of this `[TIP]` is okay or more importantly, problematic.